l10n_co_res_partner/models/l10n_co_res_partner.py

```diff
@@ -0,0 +1,459 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#                                                                             #
+# Copyright (C) 2016  Dominic Krimmer                                         #
+#                                                                             #
+# This program is free software: you can redistribute it and/or modify        #
+# it under the terms of the GNU Affero General Public License as published by #
+# the Free Software Foundation, either version 3 of the License, or           #
+# (at your option) any later version.                                         #
+#                                                                             #
+# This program is distributed in the hope that it will be useful,             #
+# but WITHOUT ANY WARRANTY; without even the implied warranty of              #
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the               #
+# GNU Affero General Public License for more details.                         #
+#                                                                             #
+# You should have received a copy of the GNU Affero General Public License    #
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.       #
+###############################################################################
+
+# Extended Partner Module
+from openerp import models, fields, api, exceptions
+from openerp.tools.translate import _
+import re
+
+
+class CountryStateCity(models.Model):
+    """
+    Model added to manipulate separately the cities on Partner address.
```

Comment:
This is already implemented in the [base_location](https://github.com/OCA/partner-contact/tree/9.0/base_location) module.